src/App.js

```diff
@@ -65,10 +67,10 @@ class App extends Component {
     return (
       <main className='main'>
         <nav className='nav-bar'>
-          {/* remember to add onClick event listener to button */}
           <button onClick={this.returnHome} className='home-button'>🍿Rancid Tomatillos</button>
         </nav>
-        {this.displayMovieDetails()}
+        {this.state.error && <h2>Sorry, there was a problem with our network</h2>}
```

Comment:
Good use of conditional rendering. To display our network errors if any appears.

Reply:
Agreed, this looks great!! As you start to see more opportunities for errors, you might even want to make an error component that takes in a dynamic message to display!